Python/Test Data Generator/README.md

```diff
@@ -0,0 +1,47 @@
+# README
+
+## Library used for the generation of fake data
+
+### Faker
+
+Faker is a Python package that generates fake data.
+
+#### Syntax
+
+faker.Faker() - Creates and initializes a faker generator, which can generate data by accessing properties named after the type of data we want.
+Each call to method fake.name() yields a different (random) result. This is because faker forwards faker.Generator.method_name() calls to faker.Generator.format(method_name).
+
+## Working of Code
+
+This code asks the user's input for the numbers of entries required and generates a fake test dataset comprising of student details enrolled in online courses.
+
+The parameters of the constructed dataset are-
+
+Student ID
+
+Student Name
+
+Email Id
+
+Phone Number
+
+Date of Birth
+
+House Address
+
+Latitude
+
+Longitude
+
+Country
+
+Course Comments
```

Comment:
Keep it as a bulleted list. And also remove a lot of blank lines on the markdown.